tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerBehavior.java

```diff
@@ -243,6 +260,70 @@ private PortalServer getPortalServer()
         return retval.toArray( new String[0] );
     }
 
+    private boolean hasAriesJmxBundlesInstalled()
```

Comment:
Check this before portal has started and if portal verison is greater than DXP SP2, add them to /osgi/static.